cmd/broadcaster-cli/app/utxos/broadcast/broadcast.go

```diff
@@ -212,4 +218,10 @@ func init() {
 		log.Fatal(err)
 	}
 
+	Cmd.Flags().Int("size-jitter", 0, "Enable the option to randomise the transaction size by adding random data and using multiple inputs, the parameter specifies the maximum size of bytes that will be added to OP_RETURN (plus OP_RETURN header if specified)")
```

Comment:
The name of the flag should be in camel case to be consistent with the other flags => `sizeJitter`

Reply:
Done